reporthandling/results/v1/resourcesresults/resultmethods.go

```diff
@@ -48,8 +47,7 @@ func (result *Result) ListControlsIDs(f *helpersv1.Filters) *helpersv1.AllLists
 func (result *Result) ListControlsNames(f *helpersv1.Filters) *helpersv1.AllLists {
 	controls := &helpersv1.AllLists{}
 	for i := range result.AssociatedControls {
-		s := result.AssociatedControls[i].GetStatus(f).Status()
-		controls.Append(s, result.AssociatedControls[i].GetName())
+		controls.Append(result.AssociatedControls[i].GetStatus(f).Status(), result.AssociatedControls[i].GetName())
 	}
 	controls.ToUniqueControls()
```

Comment:
@kooomix Notice, we run `controls.ToUniqueControls()` when listing controls names. This is incorrect since the controls names are no longer unique.

Reply:
@dwertent, yeah, it seems like the use of the `AllLists` struct with control names should be re-evaluated, depending on the usage.

Reply:
@dwertent I didn't find where we use function "ListControlsNames" - where is it being used?
If not used, we better remove the function completely.

Reply:
@kooomix Pls feel free to open a PR for this.